### PR TITLE
Remove 'bazel/$package:$tag' naming format requirements from add_apt_key, install_pkgs, and download_pkgs rules

### PR DESCRIPTION
**Pull request by ArthurRab**
Also removed now redundant macros from run.bzl

### CODE REVIEW

**Comment by dlorenc** [comment]
cc @tejal29 can you take a look at this one?

**Comment by tejal29** [comment]
Looking!

File: tests/package_managers/BUILD [hunk 1]
```diff
@@ -31,7 +31,7 @@ rule_test(
 file_test(
     name = "test_download_pkgs_docker_run",
     file = ":test_download_pkgs",
-    regexp = ".*docker run -w=\"/\" -d --privileged bazel/ubuntu:ubuntu_16_0_4_vanilla sh -c $'.*",
+    regexp = ".*ubuntu/ubuntu_16_0_4_vanilla.tar).*",
```
[the remark this answers is not in the document]
Comment (ArthurRab) [reply tests/package_managers/BUILD]: Added a space and end of line. I think that's as specific as I can make it for that line without assuming the location of the image_loader.sh file.

File: util/image_loader.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+temp=$(docker load --input $1)
```
[the remark this answers is not in the document]
Comment (ArthurRab) [reply util/image_loader.sh]: Added -o to override and an sh_test //tests/util:image_loader_test

[the remark this answers is not in the document]
Comment (tejal29) [reply util/image_loader.sh]: thanks.

File: tests/package_managers/BUILD (hunk 1 above)
[the remark this answers is not in the document]
Comment (tejal29) [reply tests/package_managers/BUILD]: what would this line look like

[the remark this answers is not in the document]
Comment (tejal29) [reply tests/package_managers/BUILD]: Probably as "sh .*<loaderscripitname> ubuntu/ubuntu_16_0_4_vanilla.tar"

[the remark this answers is not in the document]
Comment (ArthurRab) [reply tests/package_managers/BUILD]: The line it was referring to was image_name=$(sh {image_loader_path} {image_tar}). Good idea though, I added the beginning of the line too.

**Comment by tejal29** [comment]
LGTM @ArthurRab

**Pull request merged**